### PR TITLE
#1587 App intro page and App header fix

### LINKED ISSUE #1587: Mobile -- Application intro page layout
Part of Epic #1548

Image

Could do with a bit of tidying up. Reducing the padding on the outside should mostly do it, then see what else it needs

### PR DESCRIPTION
**Pull request by CarlosNZ**
Fix #1587 

Plus, I fixed up the header at the top of the application page. Turns out we'd been doing something wrong the whole time -- the "organisation" that appears at the top should be the company that made the application we're looking at, not the current logged in company (that's already in the Header bar!), so have fixed that and tidied it up a little.

WrongRight

### CODE REVIEW

File: semantic/src/site/globals/site.overrides [hunk 1]
```diff
@@ -461,15 +461,19 @@ a:hover {
...
+
 // Moves back up if UserArea is not displayed
 #application-area.non-registered {
   margin-top: -@headerHeight;
 }
 
 #content-area .top-container {
   margin-top: 20px;
-  flex-wrap: wrap;
-  justify-content: space-between;
```
Comment (CarlosNZ) [inline semantic/src/site/globals/site.overrides:472]: Flex properties weren't doing anything, as it's not a flex container 🙄

File: semantic/src/site/globals/site.overrides [hunk 2]
```diff
@@ -478,14 +482,7 @@ a:hover {
...
   font-size: @fontSmall;
   font-weight: @regularWeight;
-  padding-left: 10%;
-}
-
-// Fix for company header on mobile to not clash with 'back to applications'
-@media only screen and (min-width: @tabletBreakpoint) {
-  #content-area .heading-alt {
-    margin-top: -25px;
-  }
```
Comment (CarlosNZ) [inline semantic/src/site/globals/site.overrides:488]: Don't seem to need any more

File: semantic/src/site/globals/site.overrides [hunk 3]
```diff
@@ -498,6 +495,9 @@ a:hover {
   background-color: @surfacesWhite;
   width: 800px;
   padding: 30px 50px 32px;
+  @media only screen and (max-width: @largestMobileScreen) {
+    padding: 20px;
+  }
```
Comment (CarlosNZ) [inline semantic/src/site/globals/site.overrides:500]: This is all that's need to fix #1587

**Review by guatemartin — APPROVED** [review]
All makes sense.

**Pull request merged**